.github/workflows/build.yml

```
@@ -30,31 +30,53 @@ jobs:
+        rm -rf /tmp/.buildx-cache
+        mv /tmp/.buildx-cache-new /tmp/.buildx-cache
+    - name: Build the Docker image build environment
+      run: |
+        docker build --build-arg GECKODRIVER_VERSION=$GECKODRIVER_VERSION \
+          --build-arg HOST_TRIPLE=$HOST_TRIPLE \
+          --build-arg PLATFORM=$PLATFORM \
+          -t local/geckodriver-cross-builder .
 
-    - name: Build the geckodriver binaries in the artifacts directory
+    - name: Build geckodriver binaries in the artifacts directory
       run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:20: Double quote to prevent globbing and word splitting

(shellcheck)